doc/source/rst.rst

```diff
@@ -2,14 +2,377 @@
+* **It's standardized and uniform.** It has a fairly `comprehensive spec <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html>`_ and a single canonical implementation that is still being actively developed - the `docutils <http://docutils.sourceforge.net/>`_ project.
+* **It has built-in support for extensions.** Extension is a core design principle, and both custom roles (for inline elements) and directives (for block elements) can be easily added. It's therefore straightforward to add extensions for commonly-needed stuff like syntax-highlighted code blocks, math equations for Latex rendering and so on.
+* **reST** is the default plaintext markup language used by `Sphinx <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_, the Python Documentation Generator used by OTC Helpcenter projects.
+
+The following page will show how to work with commonly used syntax of **reST** in OTC Helpcenter projects, more comprehensive documentation can be found:
+
+* `Docutils RST`_
+* `Sphinx RST`_
+* `Create Documentation with RST, Sphinx, Sublime and GitHub`_
+
+Paragraphs
+==========
+
+The paragraph is the most basic block in a reST document. Paragraphs are simply chunks of text separated by one or more blank lines. As in Python, indentation is significant in reST, so all lines of the same paragraph must be left-aligned to the same level of indentation.
+
+Inline markup
+=============
+
+The standard reST inline markup is quite simple to use:
+
+.. code-block:: rst
+
+   one asterisk: *text* for emphasis (italics),
+
+   two asterisks: **text** for strong emphasis (boldface), and
+
+   backquotes: ``text`` for code samples.
+
+Lists and Quote-like blocks
+===========================
+
+List markup is natural, just place an asterisk at the start of a paragraph and indent properly. The same goes for numbered lists; they can also be autonumbered using a # sign:
+
+.. code-block:: rst
+
+   * This is a bulleted list.
+   * It has two items, the second
+     item uses two lines.
+   
+   1. This is a numbered list.
+   2. It has two items too.
+   
+   #. This is a numbered list.
+   #. It has two items too.
+
+.. Note:: It is also possible to nest these types of lists in another. 
+
+Headings, Sections, Subsections
+===============================
+
+Headings are created by underlining and overlining the title with a **=** character, at least as long as the text:
+
+.. code-block:: rst
+
+   =================
+   This is a Heading
+   =================
+
+Sections are created by underlining the title with a **=** character, at least as long as the text:
+
+.. code-block:: rst
+
+   This is a Section
+   =================
+
+Subsections are created by underlining the title with a **-** character, at least as long as the text:
+
+.. code-block:: rst
+
+   This is a Subsection
+   --------------------
+
```

Comment:
I would add that it is possible to go further with sub-subsections and so on by using any character reasonable for underlines. Important is that is is consequent through whole doc (not page): https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure